src/js/logger.js

```diff
@@ -28,17 +29,23 @@ RiseVision.Common.LoggerUtils = (function() {
 
       json.company_id = companyId;
       json.display_id = displayId;
-      
-      if(location.href) {
-        var parent = getParameterByName("parent", location.href); // legacy presentations
-        var type = getParameterByName("type", location.href); // templates
-        var presentation_type = type ? type : (parent ? getParameterByName("type", parent) : "");
-        var schedule_id = parent ? getParameterByName("id", parent) : "";
-        if (presentation_type ==="sharedschedule") {
-          json.presentation_type = presentation_type;
-          json.schedule_id = schedule_id;
-          json.unique_id = getUniqueId();
-        }
+
+      var parent = utils.getQueryParameter("parent"); // legacy presentations
+      var type = utils.getQueryParameter("type"); // templates
+      var env = utils.getQueryParameter("env"); // endpoint
+      var viewerId = utils.getQueryParameter("viewerId");
+
+      var presentation_type = type ? type : (parent ? utils.getQueryStringParameter("type", parent) : "");
+      var endpoint_type = env ? env : (parent ? utils.getQueryStringParameter("env", parent) : "");
+      var viewer_id = viewerId ? viewerId : (parent ? utils.getQueryStringParameter("viewerId", parent) : "");
+      var schedule_id = parent ? utils.getQueryStringParameter("id", parent) : "";
+
+      json.viewer_id = viewer_id;
+
+      if (presentation_type ==="sharedschedule") {
+        json.presentation_type = presentation_type;
+        json.schedule_id = schedule_id;
+        json.endpoint_type = endpoint_type;
```

Comment:
@stulees I wonder if calling this field the `endpoint_type` is a good idea. The way the Endpoint Type is described in the Env Verifier requirements and Logging Architecture is a combination of `type` & `env` parameters. The `env` URL parameter is sort of a subtype of the Shared Schedule type. I would just call it `env` so we don't mix terminology. What do you think?